scripts/utils.js

```diff
@@ -0,0 +1,20 @@
+const months = [
+  'January',
+  'February',
+  'March',
+  'April',
+  'May',
+  'June',
+  'July',
+  'August',
+  'September',
+  'October',
+  'November',
+  'December',
+];
+
+// eslint-disable-next-line import/prefer-default-export
+export function formatDate(date) {
+  const d = new Date(date);
+  return `${months[d.getMonth()]} ${d.getDate()}, ${d.getFullYear()}`;
```

Comment:
There is a built in Javascript functions for this:
```javascript
d.toLocaleDateString('en-US', {
      month: 'long',
      day: '2-digit',
      year: 'numeric',
})
```

and we'll need to see how to correctly display it in the geosites too.

I would maybe also not create a new JS file just for this function. For the moment, until we feel like it grew too much, we can add them to `scripts.js` to avoid an extra request.